Frontend/android-native/app/src/main/AndroidManifest.xml

```diff
@@ -44,11 +44,23 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme"
         android:usesCleartextTraffic="${isUsesCleartextTraffic}"
-        tools:replace="android:allowBackup">
+        tools:replace="android:allowBackup"
+        android:name="com.clevertap.android.sdk.Application">
 
         <meta-data android:name="com.facebook.sdk.ApplicationId" android:value="@string/facebook_app_id"/>
         <meta-data android:name="com.facebook.sdk.ClientToken" android:value="@string/facebook_client_token"/>
 
+        <meta-data
+            android:name="CLEVERTAP_ACCOUNT_ID"
+            android:value="TEST-6WK-8Z6-586Z"/>
```

Comment:
@ashkriti can you move these value to properties instead of writing it here directly